src/Components/Preview.lua

```diff
@@ -1,4 +1,5 @@
 local Selection = game:GetService("Selection")
+local CoreGui = game:GetService('CoreGui')

```

Comment:
```suggestion
local CoreGui = game:GetService("CoreGui")
local Selection = game:GetService("Selection")
```

Consistency is important!